getting_started/topics/first-boot/boot.adoc

```diff
@@ -8,7 +8,13 @@ image:{project_images}/standalone-boot-files.png[]
 
 To boot the server:
 
-.Linux/Unix
+.Linux/Unix (rpm)
```

Comment:
These docs are shared between Keycloak and RH-SSO. RPMs are only available for RH-SSO, but this change will show steps for RPMs in both sets of docs. It would be better to put a new paragraph above (after go to the bin directory of the server distribution) and wrap that in a if condition to only display for RH-SSO.